ScnSoft-Education/LeetCodeTraining/AddTwoNumbers/AddTwoNumbersSolution.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace LeetCodeTraining.AddTwoNumbers
+{
+    public class ListNode
+    {
+        public int val;
+        public ListNode next;
+        public ListNode(int val = 0, ListNode next = null)
+        {
+            this.val = val;
+            this.next = next;
+        }
+    }
+
+    public static class AddTwoNumbersSolution
+    {
+        public static ListNode AddTwoNumbers(ListNode l1, ListNode l2, int n = 0)
+        {
+            if (l1 == null && l2 == null && n == 0) return null;
```

Comment:
add {..} after if condition.
Use the same style code in all your tasks

Reply:
Brackets were added